eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -131,15 +131,6 @@ jobs:
               -RepoName "$(Build.Repository.Name)"
             pwsh: true
             workingDirectory: $(Pipeline.Workspace)
-          displayName: Create API Review
```

Comment:
Step should be named as something like `Generate API review token files`

Reply:
and trim down on the parameters